python/lsst/ts/aos/utils/m1m3/m1m3_aav.py

```diff
@@ -0,0 +1,761 @@
+# This file is part of ts_aos_utils.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["AccelerationAndVelocity"]
+
+import argparse
+import asyncio
+import logging
+import os
+import pathlib
+
+import numpy as np
+import pandas as pd
+from astropy.time import Time
+from lsst.ts.criopy import M1M3FATable
+from lsst.ts.criopy.m1m3 import AccelerationAndVelocityFitter, ForceCalculator
+from lsst.ts.idl.enums.MTM1M3 import DetailedState
+from lsst_efd_client import EfdClient
+from tqdm import tqdm
+
+tqdm.pandas()
+
+
+class AccelerationAndVelocity:
+    """Compute fir of M1M3 acceleration and velocities.
+
+    Parameters
+    ----------
+    config : pathlib.Path
+        Path to configuration files.
+    load_from : pathlib.Path
+        Optional path to load data (from hd5 file). If None, data are
+        collected/computed.
+
+    Attributes
+    ----------
+    azimuths : pd.DataFrame
+        Mount azimuth. Contains actual and demand Position, Velocity and
+        Acceleration.
+    calculated : pd.DataFrame
```

Comment:
maybe calculated_forces?

Reply:
It's both forces and moments. I made it calculated_fam